recipe/meta.yaml

```diff
@@ -24,23 +21,25 @@ requirements:
     - pip
   run:
     - python >=3.6
+    - numpy >=1.6.0
```

Comment:
It's incorrect to say that there's a runtime dependency on numpy. There are parts of pytools that use numpy, but the remainder is perfectly usable without it.

Reply:
We need to make it optional upstream then: https://github.com/inducer/pytools/blob/v2022.1.8/setup.py#L41
The way it is now it would fail the `pip check` b/c the metadata says it is a mandatory dependency.

Reply:
Oh, good point. I somehow remembered removing it, but that's clearly incorrect. Sorry. https://github.com/inducer/pytools/pull/140

Reply:
Figured out where I went wrong. I was remembering https://github.com/inducer/islpy/commit/d58416b80557a91f36759d0cac2b06a437bf8ab9. Sorry! I'll go ahead and remove the hard dependency anyway.